src/NodeChanger/IdentifierRenamer.php

```diff
@@ -0,0 +1,41 @@
+<?php declare(strict_types=1);
+
+namespace Rector\NodeChanger;
+
+use Exception;
+use Nette\Utils\Strings;
+use PhpParser\Node;
+use PhpParser\Node\Identifier;
+
+final class IdentifierRenamer
+{
+    public function renameNode(Node $node, string $newMethodName): void
+    {
+        $this->ensureNodeHasIdentifier($node);
+
+        $node->name = new Identifier($newMethodName);
+    }
+
+    /**
+     * @param string[] $renameMethodMap
+     */
+    public function renameNodeWithMap(Node $node, array $renameMethodMap): void
+    {
+        $this->ensureNodeHasIdentifier($node);
+
+        $oldNodeMethodName = $node->name->toString();
+
+        $node->name = new Identifier($renameMethodMap[$oldNodeMethodName]);
+    }
+
+    private function ensureNodeHasIdentifier(Node $node): void
+    {
+        $typeNode = Strings::after(get_class($node), '\\', 3);
+
+        if (! in_array($typeNode, [
+            'MethodCall', 'StaticCall', 'PropertyFetch', 'ClassConstFetch',
```

Comment:
Do you know `MethodCall::class` trick?
That allows you to check `get_class($node)` right away.

Reply:
Does it work within `array`s? I didn't know :scream: 

Gonna use it!

Reply:
It does, since it's only "is string in strings?".

```php
in_array(get_class($node), [MethodCall::class, StaticCall::class...]
```

For `instanceof` it doesn't work for arrays.